backend/config/settings.py

```diff
@@ -135,7 +135,7 @@
 ACCOUNT_EMAIL_VERIFICATION = None
 LOGIN_URL = "http://localhost/api/authentication/login"
 
-SITE_ID = 1 if DEBUG else 2
+SITE_ID = 2 if DEBUG else 3
```

Comment:
@sevrijss we needed to change this again since example.com was still in the database (after dumping the data). I also checked my branch and this was also the case. I remembered that you had a fix, so that example.com wouldn't be in the data anymore, but for now, changing this SITE_ID got me the expected behaviour.

Reply:
Yeah, I think it's a default from django. when I add a primary key in the datadump it stops adding `example.com` and only adds our sites.

Reply:
fixed it in data dump 👌